XMLtoMD/GuidelineXmlToMD/GuidelineXmlToMD.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Probably worth considering adding the same analyzers that are referenced in the other projects